config/accounts.go

```diff
@@ -52,6 +53,14 @@ type AccountCCPA struct {
 	ChannelEnabled     AccountChannel `mapstructure:"channel_enabled" json:"channel_enabled"`
 }
 
+type AccountPriceFloors struct {
+	Enabled           bool `mapstructure:"enabled" json:"enabled"`
+	EnforceFloorRate  int  `mapstructure:"enforce_floors_rate" json:"enforce_floors_rate"`
+	BidAdjustment     bool `mapstructure:"adjust_for_bid_adjustment" json:"adjust_for_bid_adjustment"`
+	EnforceDealFloors bool `mapstructure:"enforce_deal_floors" json:"enforce_deal_floors"`
+	UseDynamicData    bool `mapstructure:"use_dynamic_data" json:"use_dynamic_data"`
```

Comment:
Since most of the floors related config parameters are specified with default values in PRD, those existed parameters would be overwritten with JSON patch operation.